staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -294,14 +294,8 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	StorageVersionHash: {Default: true, PreRelease: featuregate.Beta},
 
-<<<<<<< HEAD
-	UnauthenticatedHTTP2DOSMitigation: {Default: true, PreRelease: featuregate.Beta},
-
-||||||| 89a4ea3e1e4
-=======
```

Comment:
In upstream, this was disabled by https://github.com/kubernetes/kubernetes/pull/121196/commits/0f33a62f9768cf7120d93b74015f5bde3df9a477.

Also, switching the gate here wouldn't be enough. If this gate is to be enabled, it also needs to be switched in `/pkg/features/kube_features.go`